pkg/model/subscribe.go

```diff
@@ -22,9 +22,7 @@ type LabelKV struct {
 type SubscribeKind = string
 
 type SubscribeTarget struct {
-	Namespace string
-	// AppName represents the target app name. Fast path for "app" label.
-	AppName string
+	NamespacedApp NamespacedApp
 	//Labels  []LabelKV
 	Kind SubscribeKind
 }
```

Comment:
The original code is more suitable, it can match with definition in proto:

``` proto
message SubscribeRequestTarget {
  string namespace = 1;
  string app = 2;
  repeated SubscribeLabelKV labels = 3;
  repeated string kinds = 4;
}
```